src/symmetric_crypto/aes_256_gcm.rs

```diff
@@ -31,6 +31,7 @@ impl Aes256Gcm {
     pub const MAX_PLAINTEXT_LENGTH: u64 = 68_719_476_704;
     /// Use a 96-bit nonce.
     pub const NONCE_LENGTH: usize = 12;
+    pub const ENCRYPTION_OVERHEAD: usize = Self::MAC_LENGTH + Self::NONCE_LENGTH;
```

Comment:
Maybe you could move this in the `Dem` trait ?

I think this constant had been removed since the DEM now takes the nonce as parameter and do not prepend it to the ciphertext. The user is in charge of managing it. For example, the nonce could be generated from a public key and the shared point like in the ECIES implementation. Therefore it is never prepended to the ciphertext. In this context this `ENCRYPTION_OVERHEAD` is not correct.

I understand why you want to add it, but I am not sure it makes sense anymore. In case you choose to revert this change, you could still add it in the Findex repository as an associated constant to the `EncryptedTable` or something like that.

Reply:
done in last commit